### DISCUSSION TITLE
multiple stacks in time-series

### DISCUSSION THREAD

**Discussion by momorientes** [grafana/grafana#37875] [Dashboards, Panels & Visualizations]
As a network operator I often use grafana to display interface graphs where Positive-Y equal packets flowing in one direction and Negative-Y equal the other direction(e.g. input vs. output packets). I already deeply miss "Transform Negative-Y", which will hopefully make a comeback in #30365. 

In the past I've added all series with negative-Y-transformation to another stack and I believe multiple stacks are required to restore the full functionality other users have described in the discussions preceeding #30365. 

Since this already was an existing feature I suppose no further description is necessary, however I'll happily provide any input if necessary.

**Comment by RichiH** [comment]
What version is this?

On Grafana Cloud v8.0.6 (68fe9e3431) I see:

![image](https://user-images.githubusercontent.com/754723/129375539-ed856734-b752-4378-8a1f-f47f7ee6550b.png)

**Comment by RichiH** [comment]
To repro/check: In the right hand bar, click on override and then on add override property at the bottom:
![image](https://user-images.githubusercontent.com/754723/129375756-41f0d40a-1a70-46cc-8982-6a905de2d506.png)

**Reply by momorientes** [reply]
seems to be an error on my side, due to the missing negative transformation and the resulting "workaround" with a negative mulitplication this broke.  Sorry for the noise.
FWIW it was v8.1.0.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
